position:
	name: "TankPositionRear"
	totalTicks: 950
control:
	name: "TankCtrlRear"
positionThreshold: 0.01
